src/systems/ec/implementations/vanilla/packages/ec-utility-typography/ec-utility-typography-print.scss

```diff
@@ -104,14 +104,48 @@
   }
 
   /*
-   * Font weight
+   * Font colour
+   */
+  @each $key, $value in $ecl-colors {
+    // Filter out yellow colours (text should not be yellow)
+    @if not str-index($key, 'yellow-') {
+      .ecl-u-type-colour-#{$key} {
+        color: $value;
+        fill: $value;
```

Comment:
not sure about this...

`fill` is for SVG, not typography

now I might argue that inline SVGs (e.g. icons) should have the same colour as the text, then I'll say that icons should use `fill: currentColor` (btw, it's already the case), so there's no need to add the `fill` property here

or am I missing something? :)